esp/scm/ws_topology.ecm

```diff
@@ -59,7 +59,17 @@ ESPStruct TpCluster
     [min_ver("1.15")] bool   HasThorSpareProcess;
     [min_ver("1.12")] ESParray<ESPstruct TpMachine> TpMachines;
 };
+
 //  ===========================================================================
+ESPStruct TpServiceTarget
+{
+    string Name;
+    string Type;
+    unsigned Port;
+    bool TLSSecure;
+    bool Public;
+    string Target;
+};
```

Comment:
How is this info. going to be used?
In particular - if not public, then it is an internal service (used only internally).

I think need to 1st be sure what we want and how this/these services are going to be used.
If it's primarily to display on an eclwatch page for example, apart from *perhaps* ops. looking at the system, I don't think it's useful to display internal services.


Not much detail in the JIRA as to how this info. would be used. Can you expand ?

Out of interest, how does bare-metal expose the esp services running on other ports ?

Reply:
A user may use the port: 8010, 8002, etc. Should a user need to know the TLS?
If we only want to expose public services, I can add the filter. Should we let Admin users know internal services?
In bare-metal mode, we display similar information on Operation/System Servers page.

Reply:
I suspect we don't want to show internal services by default, not sure in general.
If it's useful, can be acted upon, then it makes sense I think, if not, it either needs to be ignored, or shown separately somehow.
I'd just show the public services for now.